releases/release-1.31/release-notes/maps/pr-124381-map.yaml

```
@@ -0,0 +1,4 @@
+pr: 124381
+releasenote:
+  text: CEL expressions and `additionalProperties` are now allowed to be used under
+    nested quantifiers in CRD schemas
```

```suggestion
  text: Added support for CEL(Common Expression Language) expressions and `additionalProperties` to be used under nested quantifiers in CRD schemas.
```

https://github.com/kubernetes/sig-release/pull/2542#discussion_r1653210923